.travis.yml

```diff
@@ -5,6 +5,8 @@ node_js:
 install:
   - npm install gulp bower -g
   - npm install
-  - bower update
+  - bower install --production
 script:
   - gulp
+  - bower install
+  - gulp test
```

Comment:
I've started updating travis files to do this when we have `devDependencies` as it ensures we're not accidentally depending on a `devDependency` in the normal `make` too then.